src/components/common/TmSessionHardware.vue

```diff
@@ -6,11 +6,35 @@
 
     <template v-if="session.browserWithLedgerSupport">
       <div class="session-main">
-        <p v-if="session.windowsDevice" class="form-message notice">
-          {{ session.windowsWarning }}
-        </p>
         <HardwareState :loading="status === `connect` ? false : true">
-          <template v-if="status === `connect` || status === `detect`">
+          <template v-if="isWindows && !hasHIDEnabled">
+            Using a Ledger on Windows requires experimental HID support in your
+            browser.
+            <template v-if="isChrome">
+              <br />
+              <br />
+              <p>
+                Please copy this link into a new tab and 'enable' the shown
```

Comment:
can't just click the link with a blank target?

Reply:
First thing I tried ;P
Apparently doesn't work due to security rules